test/ExecutionFramework.h

```diff
@@ -48,6 +48,7 @@ using Address = util::h160;
 // The various denominations; here for ease of use where needed within code.
 static const u256 wei = 1;
 static const u256 shannon = u256("1000000000");
```

Comment:
Are shannon and gwei equivalent?

Reply:
According to https://ethdocs.org/en/latest/ether.html#denominations yes

Reply:
Should I add shannon as well?

Reply:
no, but at least make it visible in the c++ code.